helmfile/01-applications.yaml

```diff
@@ -10,6 +10,8 @@ repositories:
   url: https://vmware-tanzu.github.io/helm-charts
 - name: ingress-nginx
   url: https://kubernetes.github.io/ingress-nginx
+- name: grafana
+  url: https://grafana.github.io/helm-charts
```

Comment:
On second thought, the repository should probably be listed under service cluster specific repositories :slightly_smiling_face: